'svelte-kit' is not recognized as an internal or external command

When npm install `v1.0.2` or above, I hit this error:
```
npm ERR! 'svelte-kit' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
npm ERR! 'true' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
```
`v1.0.1` is fine.

--- a/.changeset/light-pugs-design.md
+++ b/.changeset/light-pugs-design.md
@@ -1,0 +1,5 @@
+---
+'prosemirror-dev-toolkit': patch
+---
+
+fix: update svelte-tree-view to fix silly install script #40

--- a/packages/prosemirror-dev-toolkit/package.json
+++ b/packages/prosemirror-dev-toolkit/package.json
@@ -70,7 +70,7 @@
   "dependencies": {
     "html": "^1.0.0",
     "prosemirror-model": "^1.18.3",
-    "svelte-tree-view": "^1.3.2"
+    "svelte-tree-view": "^1.3.3"
   },
   "files": [
     "src",

